Cargo.toml

```diff
@@ -20,6 +20,7 @@ name = "chrono"
 default = ["clock", "std", "oldtime", "wasmbind"]
 alloc = []
 libc = []
+winapi = ["windows-targets"]
```

Comment:
Should we really call this feature "winapi" or is it for backwards compatibility?

Reply:
It is for backwards compatibility. But I wonder if that is not overly cautious.

Reply:
Per [Hyrum's law](https://www.hyrumslaw.com/), let's err on the side of backwards compat unless there are strong reasons not to.